.gitpod.yml

```diff
@@ -0,0 +1,7 @@
+# This configuration file was automatically generated by Gitpod.
+# Please adjust to your needs (see https://www.gitpod.io/docs/config-gitpod-file)
+# and commit this file to your remote git repository to share the goodness with others.
+
+tasks:
```

Comment:
Do you want me to use `nvm` to "pin" the major version of Node.js to 16 in the `.gitpod.yml` file:

```yaml
tasks:
  - init: >
      nvm install 16 &&
      nvm use 16 &&
      yarn install &&
      yarn run build
    command: yarn run start
```

`nvm` will download the latest stable version of Node.js 16 each time the Gitpod container starts up. This takes 10 seconds or so, so it's not increasing the container start-up time much. And now the `major` version is pinned, so we don't get surprised when Gitpod starts using Node 17 for example.